### PR TITLE
releasing 1.8.5

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

regular update

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

**Comment by yurijmikhalevich** [comment]
Let's get this in before the release: https://github.com/Lightning-AI/lightning/pull/16052

**Comment by tchaton** [comment]
@Borda Would be great to get this one: https://github.com/Lightning-AI/lightning/pull/16035

**Comment by luca3rd** [comment]
https://github.com/Lightning-AI/lightning/pull/16053  would be great to include

**Comment by luca3rd** [comment]
https://github.com/Lightning-AI/lightning/pull/16054 as well

**Comment by ethanwharris** [comment]
@Borda And this one please: #16016

File: docs/source-app/workflows/run_app_on_cloud/cloud_files.rst [hunk 1]
```diff
@@ -39,6 +38,18 @@ For example, the source code directory below with the ``.lightningignore`` file
 
 A sample ``.lightningignore`` file can be found `here <https://github.com/Lightning-AI/lightning.beta/blob/master/.lightningignore>`_.
 
+If you are a component author and your components creates local files that you want to ignore, you can do:
```
Comment (carmocca) [inline docs/source-app/workflows/run_app_on_cloud/cloud_files.rst:41]: Can you revert #15818? Breaks local apps that set the argument until https://github.com/Lightning-AI/lightning/pull/16057 lands.

Comment (carmocca) [reply docs/source-app/workflows/run_app_on_cloud/cloud_files.rst:41]: Nvm. #16057 landed. I cherry-picked it

**Comment by akihironitta** [comment]
@Borda @carmocca Could you revert bacfbae #16057?

**Comment by Borda** [comment]
> @Borda @carmocca Could you revert [bacfbae](https://github.com/Lightning-AI/lightning/commit/bacfbae2cb13b176215a611e7f94b0dc2d51d39c) #16057? due to the error described in #16064

dropped from here, now just revert it from master

File: docs/source-app/workflows/run_app_on_cloud/cloud_files.rst (hunk 1 above)
Comment (carmocca) [reply docs/source-app/workflows/run_app_on_cloud/cloud_files.rst:41]: @Borda please revert #15818 too

**Comment by yurijmikhalevich** [comment]
Can we get his one in too: https://github.com/Lightning-AI/lightning/commit/3b323c842d1692ef43a10c22b4ec6ebd690d070a

**Pull request merged**